project/MimaFilters.scala

```diff
@@ -39,6 +39,8 @@ object MimaFilters extends AutoPlugin {
     ProblemFilters.exclude[DirectMissingMethodProblem]("scala.concurrent.impl.FutureConvertersImpl#P.accept"),
     ProblemFilters.exclude[IncompatibleMethTypeProblem]("scala.concurrent.impl.FutureConvertersImpl#P.andThen"),
 
+    // private[immutable] utility used by VectorMap#values, analogous to collection.AbstractIterable
+    ProblemFilters.exclude[MissingClassProblem]("scala.collection.immutable.AbstractIterable"),
```

Comment:
Does this means that if this class is missing, it won't be detected as binary incompatible ? It's ok because this class is only used internally ?

Reply:
It means the class doesn't exist before, so if it's in API, it would be incompatible. But since you asked, I'll take a second look at whether this accommodation of `VectorMap` is worth it (for the one case). I also did not track down why it required `immutable` there. In fact, `VectorMap` fought all the tweaks, so I will roll back the syntax/whitespace edits and keep just the useful comment on `nextValidField`. Thanks for your question.